compass/landice/mesh.py

```diff
@@ -1,4 +1,5 @@
 import numpy as np
+import jigsawpy
```

Comment:
Now that this has moved here, it looks like the corresponding import in `compass/landice/tests/humboldt/mesh.py`can be safely deleted.